try:
    print("a" / 3)
except:
    print("Error caught!")
